src/app/health/health.component.ts

```diff
@@ -75,4 +79,16 @@ export class HealthComponent implements OnInit, AfterViewChecked {
     this.displayedColumns = Object.keys(this.eventTable.data[0]);
   }
 
+  exportCSV() {
+    const data = this.eventTable.data.map(h =>
+      Object.entries(h).reduce((object, [ key, value ]: [ any, any ]) => {
+        key = isNaN(key) ? key : this.datePipe.transform(key, 'shortDate');
```

Comment:
A better solution (and a solution to the only recent date shown) would be to convert the date from a UNIX timestamp to the Excel number.  You could put that in **CsvService**.

```
= event.date / 86400 + 25569
```

From:
https://excelribbon.tips.net/T010849_Converting_UNIX_Date_Time_Stamps.html